docs/omnibox/query-types/error-lookups.mdx

```diff
@@ -0,0 +1,32 @@
+# Error Lookups
```

Comment:
do we need any of the files in this directory?

Reply:
Hey Alex, which files are you referring to?